requirements.txt

```diff
@@ -27,7 +27,9 @@
 -e git+https://github.com/inspirehep/python-client-sword2.git#egg=sword2
 
 # python-orcid with support for v2.0 API and XML
--e git+https://github.com/ORCID/python-orcid.git@2.0update#egg=orcid
+# -e git+https://github.com/ORCID/python-orcid.git@2.0update#egg=orcid
```

Comment:
you can remove this line, they released the new API upstream so we won't need it again.